Library/Formula/jbig2dec.rb

```diff
@@ -12,9 +10,15 @@ class Jbig2dec < Formula
     sha1 "cbb4dfe055be243427210d03304c20760fb00bd7" => :lion
   end
 
+  depends_on "automake" => :build
+  depends_on "autoconf" => :build
+  depends_on "libtool" => :build
+
   def install
-    system "./configure", "--disable-dependency-tracking", "--prefix=#{prefix}"
-    system "make install"
+    system "autoreconf", "-fvi" # error: cannot find install-sh
+    system "./configure", "--disable-dependency-tracking", "--prefix=#{prefix}",
+                          "--disable-silent-rules", "--without-libpng"
```

Comment:
```
libtool: compile:  clang -DHAVE_CONFIG_H -I. -g -O2 -Wall -c memento.c -o memento.o >/dev/null 2>&1
clang -DHAVE_CONFIG_H -I.    -DTEST -g -O2 -Wall  -c -o test_huffman-jbig2_huffman.o `test -f 'jbig2_huffman.c' || echo './'`jbig2_huffman.c
jbig2_image_png.c:29:10: fatal error: 'pngstruct.h' file not found
#include <pngstruct.h>
         ^
1 error generated.
make: *** [jbig2_image_png.o] Error 1
make: *** Waiting for unfinished jobs....
```

That file exists in the source distributions of ` libpng ` but since 1.5.0 has not been installed and made available to applications for use:

> Since libpng-1.5.0, direct access to these structs is not allowed, and the definitions of the structs
reside in private pngstruct.h and pnginfo.h header files that are not accessible to applications.  It is strongly suggested that new programs use the new APIs (as shown in example.c and pngtest.c), and older programs be converted to the new format, to facilitate upgrades in the future.